class ObjectAlreadyExistsException(Exception):
    pass


class ObjectNotFoundError(Exception):
    pass
